ShowCase/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Threading.Tasks;
+using Nimiq;
+
+namespace ShowCase
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            MainAsync().GetAwaiter().GetResult();
+        }
+
+        private static async Task MainAsync()
+        {
+            var client = new NimiqClient();
+
+            Console.WriteLine(await client.Consensus());
+
+            var accounts = await client.Accounts();
+
+            Console.WriteLine(accounts.Length);
+
+            foreach(var account in accounts)
+            {
+                Console.WriteLine(account);
+            }

```

Comment:
Would be cool if the examples can be a bit more extensive to really give a good starting point in using the cleint. Something more realistic and useful. Similar for other client libs.

Reply:
[Improved](https://github.com/rraallvv/csharp-client/blob/master/ShowCase/Program.cs) the example program